resources/metadata/cloudFoundryDeploy.yaml

```diff
@@ -280,7 +280,7 @@ spec:
           - name: cloudFoundry/mtaExtensionDescriptor
       - name: mtaExtensionCredentials
         type: "map[string]interface{}"
-        description: "Defines a map of credentials that need to be replaced in the `mtaExtensionDescriptor`. This map needs to be created as `value-to-be-replaced`:`id-of-a-credential-in-jenkins`"
+        description: "Defines a map of credentials that need to be replaced in the `mtaExtensionDescriptor`. This map needs to be created as `value-to-be-replaced`:`id-of-a-credential-in-jenkins`. The placeholders in the extension descriptor file(s) looks like: `<%= value-to-be-replaced %>`. When used outside Jenkins the secret which corresponds to `id-of-a-credential-in-jenkins` needs to be provided as environment variable in screaming snake case, e.g.: `export ID_OF_A_CREDENTIAL_IN_JENKINS=<secret>`. `id-of-a-credential-in-jenkins` needs to be provided in a way so that it can be translated into a valid environment variable name (e.g. no leading number). `value-to-be-replaced` must match this regex: `^[-_A-Za-z0-9]+$`."
```

Comment:
Is it possible for the description to occupy several lines as this would improve readability?

Defines a map of credentials that need to be replaced in the `mtaExtensionDescriptor`. 
Create the map `value-to-be-replaced:id-of-a-credential-in-jenkins` with the following placeholder in the extension descriptor file(s): `<%= value-to-be-replaced %>`. 
If used outside Jenkins, provide a secret `id-of-a-credential-in-jenkins` as an environment variable written uppercase underscore-separated letters ("screaming snake case"), for example, `ID_OF_A_CREDENTIAL_IN_JENKINS`. 
Provide `id-of-a-credential-in-jenkins` in a way so that it can be translated into a valid environment variable name (for example, no leading number). `value-to-be-replaced` must match the regular expression pattern `^[-_A-Za-z0-9]+$`.

Reply:
@LauraVeinberga :

> Is it possible for the description to occupy several lines as this would improve readability?

To my knowledge there is no example in our docu where the parameter description is html formatted. From that point of view I hesitate to introduce that. Seems to be complicated to use html formatting. E.g. we have this [step docu](https://www.project-piper.io/steps/cloudFoundryCreateService/). Here the description of the step itself is obviously provided in an html like way. But that does not work (... should be fixed by the way, but that is clearly out of scope here). In that step we have also an example for another rather long [parameter description](https://www.project-piper.io/steps/cloudFoundryCreateService/#manifestvariables) which is also not  html formatted. Other reason why I hesitate: to my understanding the description is also used for e.g. `./piper cloudFoundryDeploy --help`. Here - on the command link - html like formatting might be confusing.

> maybe it would be better to write: start with a letter/don't start with a number.

done.